integration-tests/testutils/build/build_test.go

```diff
@@ -263,3 +264,15 @@ func (s *BuildSuite) TestAssetsDoesNotBuildSnapdFromBranchIfNotInstructedTo(c *c
 		check.Commentf("Expected 0 call to execCommand with %s, got %d",
 			buildSnapdCmd, buildCall))
 }
+
+func (s *BuildSuite) TestAssetsHonoursBuildTags(c *check.C) {
+	testBuildTags := "mybuildtag"
+	Assets(s.useSnappyFromBranch, s.arch, testBuildTags)
```

Comment:
With this change I'm getting a golint error:

```integration-tests/testutils/build/build.go:68:6: func name will be used as build.BuildAssets by other packages, and that stutters; consider calling this Assets```

The only place where it is called unscoped is in the package tests, maybe we could use an alias there, like `buildAssets`, to make it clearer?